doc/install/install.md

```diff
@@ -12,20 +12,30 @@ kubectl create -f deploy/upstream/quickstart/crds.yaml
 kubectl create -f deploy/upstream/quickstart/olm.yaml
 ```
 
-## OpenShift
+## Release binary
 
-OLM is installed by default in OpenShift 4.0 and above.
+Check out the latest [releases on github](https://github.com/operator-framework/operator-lifecycle-manager/releases) for release-specific install instructions.
 
 ## Run locally with minikube
 
 This command starts minikube, builds the OLM containers locally with the minikube-provided docker, and uses the local configuration in [local-values.yaml](local-values.yaml) to build localized deployment resources for OLM.
 
-```
-make run-local
+```bash
+# Add pre-requisites for manual installation
+$ make install-crd
+
+# To install and run locally
+$ make run-local
```

Comment:
`run-local` already creates all of the required resources in a `kind` or `minikube` cluster. There's no need to apply CRDs separately.